src/assets/images/index.js

```diff
@@ -1,5 +1,18 @@
 export const images = {
+  // TODO: deprecate
```

Comment:
Since the gradient image can changed according to many different requirements I think it's better that the user will provide the image instead. 
If we want to provide defaults we can keep these images, but keep them as part of the component assets and don't expose them in our Assets

Reply:
Moved internally